src/net/tls.cc

```diff
@@ -436,12 +436,20 @@ class tls::certificate_credentials::impl: public gnutlsobj {
     client_auth get_client_auth() const {
         return _client_auth;
     }
-    void set_session_resume_mode(session_resume_mode m) {
+    void set_session_resume_mode(session_resume_mode m, const std::vector<uint8_t>& key = {}) {
```

Comment:
And here a span is more flexible than a vector.

Reply:
Seems like a huge overkill for an internal call. But sure.